lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart

```diff
@@ -586,13 +586,8 @@ class MailboxDashBoardController extends ReloadableController {
   }
 
   void setSelectedMailbox(PresentationMailbox? newPresentationMailbox) {
-    final previousMailbox = selectedMailbox.value;
-    if (previousMailbox == newPresentationMailbox) {
-      selectedMailbox.value = newPresentationMailbox;
-      selectedMailbox.refresh();
```

Comment:
why we prevent refresh in this case?

Reply:
We've removed the `currentMailboxId` variable from the ThreadView so we won't need to force the mailbox to update again, when it's selected again.